### ISSUE TITLE
Missing GitHub Release for tag `1.13.0`

### ISSUE THREAD

**Issue by sbuerk** [DeepLcom/deepl-mock#5]
For the `1.13.0` tag the corresponding  GitHub Release is missing.

That make's it hard for automation to retrieve the latest version
without checking out the whole repository every time:

```bash
gh release list --repo DeepLcom/deepl-mock \
  --json createdAt,isDraft,isLatest,isPrerelease,name,publishedAt,tagName \
  --jq '.[] | select(.isLatest) | .tagName'
```

Expected: v1.13.0
Actual: v1.12.0

Is it reasonable to create an release for the 1.13.0 tag and ensure that try to create them for future releases (tagging) ?

**Comment by JanEbbing** [comment]
Thanks for catching this, I think this was overlooked when we released v1.13.0 . I just did a release from the tag so this should be fixed now.

**Comment by sbuerk** [comment]
Thanks a lot
